llvm/lib/Target/RISCV/RISCVInstrInfoC.td

```diff
@@ -109,6 +109,21 @@ def uimm7_lsb00 : RISCVOp,
   }];
 }
 
+// A 7-bit unsigned immediate where the least significant three bits are zero.
```

Comment:
This should be defned in Mips specific file if that's the only place it is used.